.mailmap

```diff
@@ -4,5 +4,8 @@ Benjamin Klettbach <b.klettbach@gmail.com>
 BtbN <btbn@btbn.de>
 John Bradley <jrb@turrettech.com>
 HomeWorld <homeworld@gmail.com> hwdro <pdarvaru@yahoo.com>
-Michael Fabian Dirks <info@xaymar.com> <michael.dirks@xaymar.com>
+Michael Fabian Dirks <info@xaymar.com> <Xaymar@users.noreply.github.com>
+Michael Fabian Dirks <info@xaymar.com> <michael.dirks@xaymar.com> 
+Michael Fabian Dirks <info@xaymar.com>
```

Comment:
The credited name for me should be `Michael Fabian 'Xaymar' Dirks`, not just my legal name.